Allow users to submit number answers in word form, closes #243

stacietaylorcima opened the pull request:
Here is my solution to issue #243. Let me know if there are ways I can improve/simplify. I was also curious if there was a better place for some of this code - like in a helper file or something.

sinabahram commented:
A few things. I think this may be a bit overcomplicated for what we need done. Stacie, maybe you and I can walk through the code on a call today?

 

Also, can we squash/rebase the commits so that there’s only one for this PR, which seems to make sense to me for what’s being fixed.

stacietaylorcima commented:
@sinabahram I rebased and squashed. Let's definitely go over this in a call. Let me know if there are any notes I can take care of before our call so that I can save some of your time! Thanks so much!

stacietaylorcima commented:
@sinabahram Here is the updated code that's ready to go!
